docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,369 @@
+
+            ## Steps to Reproduce
+            Please provide detailed steps for reproducing the issue.
+
+                1. step 1
+                2. step 2
+                3. see the bug...
+
+            ## Additional Context
+            Please provide any relevant information about your setup. This is important in 
+            case the issue is not reproducible except for under certain conditions.
+
+                * Machine
+                * Compiler
+                * Reference other issues or PRs in other repositories that this 
+                is related to, and how they are related.
+
+            ## Output
+            Please include any relevant log files, screenshots or other output here.
+
+
+    #. `Feature Request <https://github.com/ufs-community/ufs-srweather-app/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=>`__: New features and feature enhancements fall under this category. Propose features and enhancements using the following template. Optional sections may be deleted.
+
+        .. code-block:: console
+
+            ## Description
+            Provide a clear and concise description of the problem to be solved.
+
+            ## Solution
+            Add a clear and concise description of the proposed solution.
+
+            ## Alternatives (optional)
+            If applicable, add a description of any alternative solutions or 
+            features you've considered.
+
+            ## Related to (optional)
+            Directly reference any issues or PRs in this or other repositories 
+            that this is related to, and describe how they are related.
+
+    #. `Other <https://github.com/ufs-community/ufs-srweather-app/issues/new>`__ (e.g., text only changes): Open a blank issue, and use the "Feature Request" template above as a starting point to describe the issue. 
+
+For all issue reports, indicate whether this is an issue that you plan to work on and eventually submit a PR for or whether you are merely making a suggestion. After filling out the issue report, click on "Submit new issue." 
+
+.. _GoodPR:
+
+Making a Pull Request
+---------------------------
+
+All changes to ``develop`` branches should be handled via GitHub’s “Pull Request” (PR) functionality from a feature or bug-fix branch in the developer’s fork. Developers must follow the template PR instructions displayed in :numref:`Step %s <Template>` below and provide links to relevant GitHub issue(s). They must also indicate which tests were run on which machines. 
+
+.. note::
+
+    * If the developer wants to make use of automated testing, any SRW + regional_workflow dependencies must be opened in PR's from the same user fork and branch.
+    * The Externals.cfg file should point to any dependent branches in regional_workflow (and other components, if necessary) while under review. Before being merged, these references must be updated to the appropriate hashes in the authoritative repositories.
```

Comment:
Isn't the regional_workflow its own fork? I guess this requirement is a little confusing to me, now that I think about it...

Reply:
So, regional_workflow is it's own authoritative repository, not a fork.  The requirement to point to a developers fork in the Externals.cfg file is only valid if there is a corresponding PR for regional_workflow that goes hand-in-hand with the ufs-srweather-app.  This is only necessary when a change in ufs-srweather-app is required for the PR in regional_workflow to work (maybe a new CCPP suite is being introduced and supported in regional_workflow, for example).

Reply:
Oh, I just meant that for there to be a corresponding regional_workflow PR, the developer would have had to fork the regional_workflow repository. So he would have an SRW App fork and a regional_workflow fork, since it's not possible to have a single fork containing two repos (to my knowledge). Anyway, I'll try to clarify things in the guide, and you can tell me if my update is wrong. ;)